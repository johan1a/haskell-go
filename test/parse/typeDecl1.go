package main
import "fmt";
type kustom balbal
