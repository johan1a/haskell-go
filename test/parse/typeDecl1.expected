SourceFile (Package "main") [ImportDecl [I1 "fmt"]] [TopLevelDecl1 (TypeDecl [TypeSpec "kustom" (TypeName (TypeNameIdentifier "balbal"))])]
